front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -94,33 +172,44 @@ const ContentBuilderToolbox = ({
           id="e2e-draggable-three-column"
           ref={(ref) =>
             ref &&
-            connectors.create(
-              ref,
-              <Element canvas is={ThreeColumn} id="threeColumn" />,
-              {
-                onCreate: (node) => {
-                  selectNode(node.rootNodeId);
-                },
-              }
-            )
+            connectors.create(ref, <ThreeColumn />, {
```

Comment:
Why should this open on create? There are no settings for this component.

Reply:
This was a request from Ben actually - he wanted it to be consistent for all components. I guess the delete button is still shown at least?

Reply:
Ah, alright then 👍